python/sandbox.config

```diff
@@ -1,5 +1,5 @@
 [sdk]
-workflow_packages=single_step,multi_step_linear
+workflow_packages=single_step,multi_step_linear,workflows
```

Comment:
I don't get this line, there's no workflows directory right?

Reply:
`Get started` section in Flyte doc instructs user to create a new workflow under this folder and links to an old commit where this folder was in config file. The doc doesn't say user needs to modify the config file, so they will be running into trouble. We can either revert this change or fix doc I think.